src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -129,6 +150,27 @@ protected override void Dispose(DisposeTypes type)
             base.Dispose(type);
         }
 
+        /// <summary>
```

Comment:
This does not need. Please remove it.

Reply:
Thank you for your review!
The property Style has been removed.

Do you mean that the property ViewStyle works instead of this property?

Reply:
Yes. NUI decided not to open public `Style` for all component. Existing ones should be removed next ACR.